runtime/src/main/java/org/corfudb/runtime/exceptions/TransactionAbortedException.java

```diff
@@ -30,6 +33,17 @@
     @Getter
     AbstractTransactionalContext context;
 
+    /** True, if it is known that this abort was not caused by a false conflict. */
+    @Getter
+    @Setter
+    boolean precise = false;
+
+    /** If available, the address where the conflict was detected,
+     *  otherwise, Address.NON_EXIST.
+     */
+    @Getter
+    long conflictAddress = Address.NON_EXIST;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "conflictAddress" field final. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1165)